Lib/test/test_reprlib.py

```diff
@@ -318,6 +318,8 @@ class bar:
         # Module name may be prefixed with "test.", depending on how run.
         self.assertEqual(repr(bar.bar), "<class '%s.bar'>" % bar.__name__)
 
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure
```

Comment:
will this regressions be fixed in this PR? module name looks like not to be properly saved.

Reply:
Thank you for detailed explanation!